### ISSUE TITLE
Code Signing executable FAH programs before building installer

### ISSUE THREAD

**Issue by Hou5e** [FoldingAtHome/fah-client-bastet#62]
Enhancement: All the executable files distributed in the installer should be signed with the code-signing certificate as part of the build process for Windows.

This will prevent Anti-virus & Anti-malware software from potentially flagging the executable files as malicious. It also allows the programs to be verified by the user for not having been modified or tampered with.
(copy from: https://github.com/FoldingAtHome/fah-client-win-installer/issues/16)

**Comment by hc970** [comment]
> Enhancement: All the executable files distributed in the installer should be signed with the code-signing certificate as part of the build process for Windows.
> 
> This will prevent Anti-virus & Anti-malware software from potentially flagging the executable files as malicious. It also allows the programs to be verified by the user for not having been modified or tampered with. (copy from: [FoldingAtHome/fah-client-win-installer#16](https://github.com/FoldingAtHome/fah-client-win-installer/issues/16))

Agreed. I would almost go as far as stating this an essential requirement now. I recently installed a piece of paid for software and it wasn't signed. The installer was initially blocked. Once installed Windows Defender on Windows 11 blocked its use and quarantined the now installed files. They weren't malware just unsigned, very new and not often used.

Another AV vendor on my other system did likewise for the same software. I had to un-quarantine the software and make exceptions for it on both systems to resolve the issue.  A non-technical user would not be able to use the software in this case.

Please also consider meeting the stricter Authenticode guidelines set by Microsoft:

https://msrc.microsoft.com/update-guide/vulnerability/CVE-2013-3900

https://learn.microsoft.com/en-us/security/trusted-root/program-requirements

Thank you.
